src/extension/inspire/src/main/java/org/geoserver/inspire/ViewServicesUtils.java

```diff
@@ -7,11 +7,19 @@
 import static org.geoserver.inspire.InspireSchema.COMMON_NAMESPACE;
 import static org.geoserver.inspire.InspireSchema.VS_NAMESPACE;
 
+import org.geoserver.ExtendedCapabilitiesProvider;
 import org.geoserver.ExtendedCapabilitiesProvider.Translator;
+import org.geoserver.ows.Dispatcher;
 import org.xml.sax.helpers.NamespaceSupport;
 
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
 public final class ViewServicesUtils {
 
+    public static final String DEFAULT_SUFFIX="(default)";
```

Comment:
Since this is new, what is going to happen for a data directory that already has the INSPIRE plugin configured, on upgrade?

Reply:
fixed, the previous language field has been mantained as the default one. No backward compatibility issue should arise.